programs/helium-entity-manager/src/instructions/issue_entity_v0.rs

```diff
@@ -146,16 +158,23 @@ pub fn handler(ctx: Context<IssueEntityV0>, args: IssueEntityArgsV0) -> Result<(
     ctx.accounts.maker.name.as_bytes(),
     &[ctx.accounts.maker.bump_seed],
   ]];
-  let asset_id = get_asset_id(
-    &ctx.accounts.merkle_tree.key(),
-    ctx.accounts.tree_authority.num_minted,
-  );
 
   let name = animal_name.to_string();
+  let mut uri = format!("{}/{}", ENTITY_METADATA_URL, key_str);
+
+  // HACK: Handle wifi hotspots with v1 uri. TODO: Find a way to do this for all hotspots after migration
+  // https://docs.google.com/document/d/1Q6VdcDA3McehQYb1MzWL8Iy421D84T7UrmJE7Z9Vc_Y/edit?usp=sharing
```

Comment:
If the wifi hotspots have a key size of 1024 bytes and they are included in the args, then won't this instruction fail due to transaction size?

Reply:
Yeah, iirc the base58(check) encoding of a RSA key is like 338 bytes.. add the URL overhead to that I guess.. 

Example: https://github.com/helium/helium-crypto-rs/blob/main/src/rsa/mod.rs#L238